### ISSUE TITLE
[RSVP-8] Edit event

### ISSUE THREAD

**Issue by Thaanu2001** [fossnsbm/fossnsbm-web-backend#9]
As an admin or super admin, I should be able to edit an event if there is something that needs to change. (Ex: - If the date was postponed, I should be able to edit the date of the event.)

Blocked by #10 

#### User types ####
- Admin
- Super admin
